lib/tasks/fake_data/journeys.rb

```diff
@@ -177,12 +118,8 @@ def create_redirect_move_event(timestamp, notes, location)
       end
 
       def random_event
-        case rand(1..6)
+        case rand(1..2)
```

Comment:
how about `if rand(0..1) == 1`

Reply:
actually -just delete this `random_event()` method and move the if up into the main method ^

Reply:
We mock this method in the test so requires updating. Leaving as is painful/not my main priority atm. Will likely remove this/refactor later.